packages/backend/src/services/SavedChartsService/SavedChartService.ts

```diff
@@ -214,8 +219,14 @@ export class SavedChartService {
                 "You don't have access to the space this chart belongs to",
             );
         }
+        const views = await this.analyticsModel.countChartViews(savedChartUuid);
 
-        return savedChart;
+        await this.analyticsModel.trackChartView(savedChartUuid, user.userUuid);
```

Comment:
@ZeRego I know we talked about unifying analytics and rudderstack, but I don't see a way to add any abstraction without overcomplicating things... any ideas ? Or shoud we leave it like this for now..